packages/jest-runtime/src/index.ts

```diff
@@ -1053,6 +1055,15 @@ class Runtime {
       }),
     ];
 
+    if (!this._mainModule) {
+      this._mainModule = module;
```

Comment:
only if `module.filepath === this._testPath` or whatever

Reply:
So should I make it only `if (module.filepath === this._testPath)` or both `!this._mainModule` and this?

Reply:
both 👍